src/python/packages/amwg/amwg.py

```diff
@@ -1252,7 +1252,29 @@ class amwg_plot_set5(amwg_plot_set5and6):
     normally a world map will be overlaid. """
     name = '5 - Horizontal Contour Plots of Seasonal Means'
     number = '5'
-    
+    def customizeTemplates(self, templates, var=None):
+        """Theis method does what the title says.  It is a hack that will no doubt change as diags changes."""
+        (cnvs1, tm1), (cnvs2, tm2) = templates
+        import pdb
+        if hasattr(var, 'RMSE'):
+            RMSE = round(var.RMSE, 2)
+            CORR = round(var.CORR, 2)
```

Comment:
Clearly this is not perfect. I should put a try block where they get computed also. Once Jonathas' branch is merged we'll need to revisit this.